functions/v2/ocr/main_test.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 Google LLC.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import concurrent.futures
+import json
+
+from cloudevents.http import CloudEvent
+import mock
+
+
+import main
+
+
+class TestGCFPyOCRSample:
```

Comment:
It seems like the output isn't really verified by the tests but simply runs the methods being implemented. Is that hard to achieve, or are there other things happening in the background that is verifying the tests that I'm not aware of?

Reply:
No, but I left the test as unchanged as possible as this is a migration, not a fix. I'd normally expect the test to hit the real services, not just mocks of them, but that should probably be a future issue.

Reply:
Thanks for the explanation! If it's not already scoped, could you either file an issue on GitHub or internally with the team to look after them?